lib/dl_api_lib_testing/dl_api_lib_testing/connector/data_api_suites.py

```diff
@@ -472,3 +474,74 @@ def test_basic_preview(
         preview_rows = get_data_rows(preview_resp)
         min_preview_row_cnt = 10  # just an arbitrary number
         assert len(preview_rows) > min_preview_row_cnt
+
+
+class DefaultConnectorDataCacheTestSuite(StandardizedDataApiTestBase, RegulatedTestCase):
+    data_caches_enabled = True
```

Comment:
we're probably should check that it's enabled in the actual test, to prevent overrides from the base classes